src/hooks/useLoadTheme.js

```diff
@@ -19,7 +25,7 @@ const useLoadTheme = () => {
     if (!storedThemeKey) {
       setCurrentTheme(defaultTheme, false);
     } else {
-      if (storedThemeKey === "Custom") {
+      if (!storedCustomTheme) {
```

Comment:
Should be the other way round, no?
```suggestion
      if (storedCustomTheme) {
```